{"type": "error", "data": "Error running pipeline!", "stack": "Traceback (most recent call last):\n  File \"/home/logan/GRAG/graphrag/graphrag/index/run.py\", line 310, in run_pipeline\n    workflow_name: str = workflow.name\n                         ^^^^^^^^^^^^^\n  File \"/home/logan/GRAG/env/lib/python3.11/site-packages/datashaper/workflow/workflow.py\", line 160, in name\n    return self._schema.get(\"name\", \"Workflow\")\n  File \"/usr/lib/python3.11/bdb.py\", line 94, in trace_dispatch\n    return self.dispatch_return(frame, arg)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.11/bdb.py\", line 156, in dispatch_return\n    if self.quitting: raise BdbQuit\n                      ^^^^^^^^^^^^^\nbdb.BdbQuit\n", "source": "", "details": null}
